.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- `[OPTIONAL]` Please always make `inline-comments` for the issues or problems so that a student can understand the issue very well and make improvements. Always congratulate the student in your review summary about their accomplishments. You can write the highlight in the review summary like below example.

![image](https://user-images.githubusercontent.com/66476329/235451726-feb33327-42c6-47f1-8c73-d983fc53e713.png)